src/test/java/org/jitsi/meet/test/base/ParticipantOptions.java

```diff
@@ -0,0 +1,148 @@
+package org.jitsi.meet.test.base;
+
+import org.jitsi.meet.test.util.*;
+
+import java.util.*;
+
+public class ParticipantOptions
+    extends EnumMap<ParticipantOptions.Options, Object>
+{
+    public static final String FAKE_AUDIO_FNAME_PROP
+        = "jitsi-meet.fakeStreamAudioFile";
+
+    public static final String FAKE_VIDEO_FNAME_PROP
+        = "jitsi-meet.fakeStreamVideoFile";
+
+    private String configPrefix;
+
+    public static enum Options
```

Comment:
Most of those options are web specific and we can not have enum subclasses. We need to figure this out in some other way... I was thinking that options should be backed up by a Properties object and we'd have ParticipantOptions subclasses that declare keys ?

Reply:
Also I was thinking we should have WebParticipantBuilder like there are mobile ones and it would provide default values. Plus the loading should happen in ParticipantFactory which has the config object and it could read from there instead of System.getProperty (which is contained in the config).

Reply:
Or a WebParticipantFactory would provide default WebParticipantOptions. It's kind of hard to guess the details without trying that out.